### ISSUE TITLE
DEPR: ArrayManager

### ISSUE THREAD

**Issue by rhshadrach** [pandas-dev/pandas#55043]
Discussed in Basel, I believe there is a consensus to deprecate the ArrayManager.

cc @pandas-dev/pandas-core

**Comment by kshitizfrompali** [comment]
@rhshadrach Could you please elaborate this issue?

**Comment by Dr-Irv** [comment]
> Discussed in Basel, I believe there is a consensus to deprecate the ArrayManager.

Can you summarize that discussion here for those of us who weren't in Basel?

**Comment by rhshadrach** [comment]
I believe the main points are:

 - We do not see ArrayManager and BlockManager existing side-by-side in perpetuity. We need to choose one or the other.
 - Support for zero-copy from NumPy requires blocks, and is important to packages like scikit-learn
 - ArrayManager is not tested on the CI and is currently failing (I'm seeing 1038 failing)
 - We would like to instead pursue creating 2D EAs

**Comment by topper-123** [comment]
I agree with the points you made, especially that we should pick one or the other and I'm fine with BlockManager being the one used going forward.

I'm +1 on this.

**Comment by simonjayhawkins** [comment]
in the roadmap we have a section on "Block manager rewrite".

Now the roadmap pre dates the PDEP process and in the section "Roadmap points pending a PDEP" we have 

> pandas is in the process of moving roadmap points to PDEPs (implemented in August 2022). During the transition, some roadmap points will exist as PDEPs, while others will exist as sections below.

Now that this issue is closed do we need to make any changes to the roadmap on the web page?
